app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -281,13 +282,54 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
             actions: ["transcribe:StartStreamTranscriptionWebSocket","transcribe:StartMedicalStreamTranscription","comprehendmedical:InferICD10CM","comprehendmedical:InferRxNorm","comprehendmedical:DetectEntitiesV2"]
           })
         );
+        
+        // Dynamodb
+        const TableSessions = new ddb.Table(this, 'Table1', {
```

Comment:
Could you rename Table1 to MedicalSessions or similar? This works as the resource name which also influences how the table Name shows up on CloudFormation